### Task 1  

In [58]:
NYK_time =22
differ = 6
France_time = (NYK_time + differ)%12

In [59]:
print("France_time = ",France_time,"am")

France_time =  4 am


interpretation - Here is the time of france ,which helps matt to wake up early

### Task 2 

In [6]:
#1Euro = 1.1Dollar

In [85]:
service_charge_percentage = 1.75/100
Euros = 500
exchange_rate = 1.1

In [86]:
dollars = Euros*exchange_rate
service_charge = service_charge_percentage * dollars



In [87]:
Amount_in_hand = dollars + service_charge
print("the number of dollars spend to get 500 Euros :",Amount_in_hand)

the number of dollars spend to get 500 Euros : 559.625


interpretation - in this program , i have calculated the expense of Matt ,where he converted dollars into euros

### Task 3 

In [66]:
#no_of_days_in_france = 45days
# if per month contain 30 days then 15 days from thr next month

dollars_per_month = 4800

dollars_for_45days = dollars_per_month+ (dollars_per_month)*15/30


In [67]:
print(dollars_for_45days)

7200.0


Interpretation - Matt need 7200 dollars to spend in france for 45 days

### Task 4 

In [43]:
invested_amount= 20000
#interest = 5per
profit_after_1_month = invested_amount*0.05

In [50]:
print("the profit amount for matt:",profit_after_1_month,"$")

the profit amount for matt: 1000.0 $


Interpretation - Here is amount of profit, that  matt has invested in stocks for 1 month

### Task 5 

In [47]:
#1 kilometer = 0.602 miles

tourist_walks = 12
miles = 0.602

Matt_should_walk = tourist_walks*miles

In [49]:
print("In NYK Matt should walk :",Matt_should_walk,"miles")

In NYK Matt should walk : 7.224 miles


Interpretation - In NYK Matt should walk upto 7.224 miles for to explore i france 

### Task  6 

In [6]:
#1 kilometer = 0.602 miles
kilometer = 3328
mile = 0.602

cycling_kilometer_in_miles = kilometer*mile/1.2

In [8]:
print("the total no of laps :",cycling_kilometer_in_miles,)

the total no of laps : 1669.5466666666666


Interpretation - The cycle race will take upto 1669.54 Laps

### Task 7 

In [14]:
one_lap = 2  # time to complete one lap
break_after_laps = 5  # after 5 laps, 10 min break and also lunch break for 1 hour
total_time_hours = 10  # total time for biking in hours
break_time_minutes = 10
lunch_break = 1  # time for lunch break = 1 hour
total_distance = 3328  # total distance in laps

total_biking_time_minutes = (total_distance * one_lap)  # Corrected mistake

# Calculate the number of breaks
no_of_breaks = total_distance // break_after_laps

total_break = no_of_breaks * break_time_minutes  # Calculate total break time in minutes

# Calculate lunch break time minutes
lunch_break_time_minutes = lunch_break * 60

# Calculate total timing including lunch breaks and breaks
total_break_time_in_minutes = total_biking_time_minutes + total_break + lunch_break_time_minutes

# Calculate the number of days needed
num_days = total_break_time_in_minutes / (total_time_hours * 60)  # Corrected mistake

print("number_of_days_needed:", num_days)


number_of_days_needed: 22.276666666666667


Interpretation - the no of days to complete the kilometer of 3328 is 22.27 days

### Task 8 

In [8]:
# 1 oz = 30 ml
bottle_size = 750
oz = 30
days = 15
per_night = 30

In [9]:
juice_quantity = per_night*15
no_of_bottles = (juice_quantity*10)//bottle_size

In [10]:
juice_quantity
no_of_bottles

6

Interpretation - Matt need 6 bottles of grape juice for 2 weeks